Code/Tools/AssetProcessor/native/ui/ProductDependencyTreeModel.cpp

```diff
@@ -0,0 +1,358 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ProductDependencyTreeModel.h"
+#include "ProductDependencyTreeItemData.h"
+#include "AssetTreeItem.h"
+#include "ProductAssetTreeItemData.h"
```

Comment:
the coding standard is to prefer the include <> over include "" when possible
would it be possible to use "include<>" instead?

Reply:
I found that existing files, such as [SourceAssetDetailsPanel.cpp](https://github.com/aws-lumberyard-dev/o3de/blob/sven_ProductAssetDetail/Code/Tools/AssetProcessor/native/ui/SourceAssetDetailsPanel.cpp), ProductAssetTreeModel.cpp ...etc, also includes headers in the same directory in this manner. Could it be actually a proper way? Or perhaps these includes have to be changed as well?

Reply:
changed to angle bracket